tchannel/peer_heap.py

```diff
@@ -31,11 +33,26 @@ class PeerHeap(HeapOperation):
 
     def __init__(self):
         self.peers = []
+        self.order = 0
 
     def size(self):
         return len(self.peers)
 
     def lt(self, i, j):
+        """Compare the priority of two peers.
+
+        Primary comparator will be the score of each peer. If the score is
+        same then use FIFO policy. The ``order`` attribute of the peer tracks
+        the heap push order of the peer. This help solve the imbalance
+        problem caused by randomization when deal with same score situation.
```

Comment:
Good work on adding the documentation but the information on the specifics of how the peer heap behaves should be in the class doc, not individual method docs.